GNUmakefile

```diff
@@ -5,6 +5,12 @@ chip=amd64
 ifeq ($(shell uname -m), arm64)
   chip=arm64
 endif
+ifeq ($(shell uname -s),Linux)
```

Comment:
What is this for? Does it automatically recognize your os?

Reply:
I had problem with the `make dev` command because I run my local env on `linux`. Since the script use `darwin` as OS, I had to include the `linux` option and I added a simple logic to detect it. Please, let me know if it still works on `darwin`